rkt/rkt.go

```diff
@@ -221,7 +228,9 @@ func main() {
 	// cmdRkt.GenBashCompletionFile("dist/bash_completion/rkt.bash")
 	// os.Exit(0)
 
-	cmdRkt.Execute()
+	if err := cmdRkt.Execute(); err != nil && cmdExitCode == 0 {
```

Comment:
Also really it would be nice to handle all non-nil errors (I know we were previously just ignoring but since you're going part of the way there..)